css/style.css

```diff
@@ -0,0 +1,17 @@
+#img {
+    display: block;
+}
+
+#image {
+    width: 100%;
+    max-width: 100%;
+    height: auto;
+}
+
+.center {
+    display: block;
+    margin-left: auto;
+    margin-right: auto;
+    text-align: center;
+    max-width: 100%;
+}
```

Comment:
- [ ] Required: Please remove extra indentation and lines in your **CSS** file. Following CSS best practices will make your code clean and more organized. Please refer to the **[HTML & CSS Best Practices](https://github.com/microverseinc/curriculum-html-css/blob/main/articles/html_css_best_practices.md)** guidelines.